core/src/main/java/by/it/academy/homework/lesson3/info.md

```diff
@@ -1,62 +1,41 @@
 **Программа сделок**
 
```

Comment:
Please, group classes by logic validation/deal and place them into different packages.